CONTRIBUTING.md

```diff
@@ -13,3 +13,54 @@ If you have a question, issue, or bug, you can open a Github issue. First, make
 	* Include a code sample or an executable test case demonstrating the expected behavior that is not occurring when possible
 * **Feature / Improvement Request**: For situations in which the library or a component are working as expected, but you desire additional functionality. This is the appropriate template for requesting new components.
 * **Question**: For situations in which you need help from the BigCommerce engineering team, regarding guidance on using the BigDesign library, specific components, or other issues.
+
+## New Features/Changes
+
+Before you put signifcant effort into a new feature or API changes, make sure you have [filed an issue](https://github.com/bigcommerce/big-design/issues/new?assignees=&labels=&template=feature---improvement-request.md&title=). This allows us to have a conversation around the subject and come to an agreement around the proposed solution.
+
+If you change just fixes a bug, you can submit a pull request but ensure there is a [bug issue](https://github.com/bigcommerce/big-design/issues/new?assignees=&labels=&template=bug_report.md&title=) attached to your pull request.
+
+## Your First Pull Request
+
+A good first place to start is our list of [good first issues](https://github.com/bigcommerce/big-design/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+Once you are ready to start working on an issue, follow steps in **[Opening a Pull Request](#opening-a-pull-request)** and **[Development Workflow](#development-workflow)**.
+
+## Opening a Pull Request
```

Comment:
@chanceaclark Do you want to link to the pull request template that you added in another PR to this? I think it would be good to do so.

Reply:
I added it here https://github.com/bigcommerce/big-design/pull/536

It automatically prefills the PR descriptions so leaving a link out if fine.